pkgs/applications/networking/p2p/transgui/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchsvn, pkgconfig, makeDesktopItem, unzip, fpc, lazarus,
+libX11, glib, gtk, gdk_pixbuf, pango, atk, cairo, openssl }:
+
+stdenv.mkDerivation rec {
```

Comment:
@RamKromberg Looks like version 5.12.0 has been released on GitHub, see [here](https://github.com/transmission-remote-gui/transgui/releases). I'm unfamiliar with fpc and lazarus. I therefore don't feel confident in updating this, so I was wondering if you could if you have the time and inclination. I use transgui on a regular basis, so I thought it'd be nice if we could ship the latest version. In any case, thank you!